lib/ansiblelint/rules/IncludeMissingFileRule.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020, Joachim Lusiardi
+# Copyright (c) 2020, Ansible Project
+
+from __future__ import print_function
+from ansiblelint import AnsibleLintRule
+import os.path
+import ansible.parsing.yaml.objects
+
+
+class IncludeMissingFileRule(AnsibleLintRule):
+    id = '4711'
+    shortdesc = 'an include task references a non existent file'
+    description = 'Tabs can cause unexpected display issues, use spaces'
+    severity = 'HIGH'
+    tags = ['formatting']
+    version_added = 'v4.2.0'
```

Comment:
It's already released so it's probably `v4.3.0` if this gets in before we cut the release.

Reply:
ok, i'll change that